jekyll/_cci2/dedicated-hosts-macos.adoc

```diff
@@ -0,0 +1,92 @@
+---
+version:
+- Cloud
+---
+= Dedicated Hosts for macOS
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn about CircleCI's dedicated host offering for macOS users.
+:icons: font
+:toc: macro
+:toc-title:
+
+== Dedicated Hosts for macOS
+
+The dedicated host resource class is a new macOS option for those developing, building, testing, and signing iOS, iPadOS, macOS, WatchOS, and tvOS applications using the Xcode IDE. These dedicated resources provide an isolated environment for increased security.
+
+This resource class requires a 24-hour minimum lease and runs on Intel-based Mac hardware.
+
+The identifier for the dedicated host resource is metal and supports the following Xcode images:
+
+- Xcode 13.0.0
+- Xcode 12.5.1
+- Xcode 12.4.0
+- Xcode 12.3.0
+- Xcode 12.2.0
+
+=== Pricing and specs
+
+Once a dedicated host has been allocated, you have exclusive access to it for a minimum of 24 hours. If the dedicated host is already in use when a job is kicked off, an additional dedicated host is reserved (with its own 24 hour lease window).
+
+Each account can currently have a maximum of three concurrent dedicated hosts. Any time over the initial 24 hours that a dedicated host is in use will be charged a per-minute overage rate (see table below for pricing details).
+
+[.table.table-striped]
+[cols=5*, options="header", stripes=even]
+|===
+| Resource Class Name
+| vCPU
+| Memory
+| Storage
+| Cost
+
+| `metal`
+| 12 
+| 32 GB
+| 200 GB
+| 00 credits per minute (24-hour minimum)
+|===
+
+=== Known limitations
+
+- The resource class does not currently support test splitting or parallelism.
+- The host gets cleaned between jobs, which can take 5-45 minutes currently.
```

Comment:
I would rephrase this so the sentence doesn't end on "currently." Maybe: "The host gets cleaned between jobs, which can currently take between 5-45 minutes."